Tests/SonarScanner.MSBuild.PreProcessor.Test/JreCaching/JreCacheTests.cs

```diff
@@ -60,7 +63,7 @@ public void CacheDirectoryIsCreated()
         directoryWrapper.Exists(cache).Returns(false);
         var fileWrapper = Substitute.For<IFileWrapper>();
 
-        var sut = new JreCache(directoryWrapper, fileWrapper);
+        var sut = new JreCache(testLogger, directoryWrapper, fileWrapper);
```

Comment:
What do you think about extracting this to a method, so that we have to alter with it in one place only?
We do something like this for SQ/SC web server tests, and I did a similar cleanup.
Something like this:

```csharp
    private SonarCloudWebServer sut;

    public SonarCloudWebServerTest()
    {
        downloader = Substitute.For<IDownloader>();
        version = new Version("5.6");
        logger = new TestLogger();
    }

    [TestInitialize]
    public void Init() =>
        sut = CreateServer();

    private SonarCloudWebServer CreateServer(IDownloader webDownloader = null, IDownloader apiDownloader = null, HttpMessageHandler handler = null)
    {
        webDownloader ??= downloader;
        apiDownloader ??= Substitute.For<IDownloader>();
        return new SonarCloudWebServer(webDownloader, apiDownloader, version, logger, Organization, httpTimeout, handler);
    }
```

Reply:
I will address this later when the tests are more complete.